src/main/controllers/TribunalOrderOrRequestDetailsController.ts

```diff
@@ -25,6 +25,7 @@ export default class TribunalOrderOrRequestDetailsController {
     if (selectedRequestOrOrder.value.notificationState !== HubLinkStatus.VIEWED) {
       try {
         await updateSendNotificationState(req, logger);
+        selectedRequestOrOrder.value.notificationState = HubLinkStatus.VIEWED;
```

Comment:
Are you saving this change back to the DB at any point? should probably call save hub status

Reply:
Nope this one is not needed I will close the PR